ext/neural_snow/DataTools.jl

```diff
@@ -542,6 +633,1171 @@ function snotel_metadata(;
         link = link * field * ","
     end
     data = CSV.read(HTTP.get(link).body, DataFrame, comment = "#", delim = ",")
+    data[!, :Elevation] .= data[!, :Elevation] * 12 * 2.54 / 100
+    return data
+end
+
+
+"""
+    fix_temp(temps; alaska)
+
+Apply temperature bias corrections to raw SNOTEL temperature data, using temperature correction
+formula developed by Atwood et. al. (2023)
+
+# Arguments
+- `temps::Vector{<:Real}`: the temperature data to fix.
+- `alaska::Bool`: Specifies whether to apply the Alaska fix (as opposed to the CONUS fix). Default
+ is `false`.
+"""
+function fix_temp(temps::Vector{<:Union{Real, Missing}}; alaska::Bool = false)
+    tc = alaska ? (temps .+ 67.296) ./ 197.278 : (temps .+ 65.929) ./ 194.45
+    c1 = 610558.226380138
+    c2 = -2056177.65461394
+    c3 = 2937046.42906361
+    c4 = -2319657.12916417
+    c5 = 1111854.33825836
+    c6 = -337069.883250001
+    c7 = 66105.7015922199
+    c8 = -8386.78320604513
+    c9 = 824.818021779729
+    c0 = -86.7321006757439
+
+    new_t =
+        c1 .* (tc .^ 9) .+ c2 .* (tc .^ 8) .+ c3 .* (tc .^ 7) .+
+        c4 .* (tc .^ 6) .+ c5 .* (tc .^ 5) .+ c6 .* (tc .^ 4) .+
+        c7 .* (tc .^ 3) .+ c8 .* (tc .^ 2) .+ c9 .* tc .+ c0
+
+    return new_t
+end
+
+"""
+    serreze_qc(input, id, state)
```

Comment:
is this specific to SNOTEL data? Also for yan_qc?

Reply:
serreze's and yan's filter is specific to SNOTEL sites/data.